packages/relay-compiler/core/RelaySchemaUtils.js

```diff
@@ -214,6 +214,14 @@ function assertTypeWithFields(type: GraphQLType): GraphQLObjectType | GraphQLInt
   return type;
 }
 
+// TODO: Remove this when flow types for `getNullableType` in graphql-js
```

Comment:
Yeah - I think this is an existing real error in the Relay compiler source code which you're just exposing via this change.

You can work around it by replacing `type` with `(type: $FlowFixMe)` where the errors are emerging from